pennylane/ops/qubit/parametric_ops.py

```diff
@@ -59,10 +59,22 @@ class RX(Operation):
     def num_params(self):
         return 1
 
-    @classmethod
-    def _matrix(cls, *params):
-        theta = params[0]
+    @staticmethod
+    def compute_matrix(theta):
+        """Canonical matrix representation of this operator.
 
+        Args:
+            theta (tensor_like or float): rotation angle
+
+        Returns:
+            tensor_like: canonical matrix
+
+        **Example**
+
+        >>> qml.RX.compute_matrix(torch.tensor(0.5))
+        tensor([[0.9689+0.0000j, 0.0000-0.2474j],
+                [0.0000-0.2474j, 0.9689+0.0000j]])
```

Comment:
I decided to use tensors in some of the examples. Ideally I could use different frameworks, but I was too lazy.

Reply:
I do the same as well lol